### ISSUE TITLE
Add additional features to home page

### ISSUE THREAD

**Issue by stevenwjy** [nuansa-cp/gapura#28]
- Past NUANSA preview (last 4 shows only)
- Sponsors
- FAQ
- Buy tickets
- Contact Us

**Comment by stevenwjy** [comment]
For now, can just hardcode everything ._.

After #35 is done, can query to database for retrieving info

**Comment by stevenwjy** [comment]
Ok, solved in #37.
Only need to fix the view according to the design provided by the pubs team, will open another issue for that later.
